pkg/ebpf/bytecode/asset_nobindata.go

```diff
@@ -0,0 +1,20 @@
+// +build !ebpf_bindata
+
+package bytecode
+
+import (
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// GetReader returns a new AssetReader for the specified file asset
+func GetReader(name string) (AssetReader, error) {
+	asset, err := os.Open(path.Join(DefaultBPFDir, path.Base(name)))
+	if err != nil {
+		return nil, errors.Wrap(err, "could not find asset")
```

Comment:
Does the error include the path that we searched at? That might be useful for debugging bad configurations

Reply:
Yes, it does